packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -15,37 +25,56 @@ import {
     DeviceModel,
 } from '@trezor/device-utils';
 
-export type FirmwareAction =
-    | {
-          type: typeof FIRMWARE.SET_UPDATE_STATUS;
-          payload: ReturnType<GetState>['firmware']['status'];
-      }
-    | { type: typeof FIRMWARE.SET_HASH; payload: { hash: string; challenge: string } }
-    | { type: typeof FIRMWARE.SET_HASH_INVALID; payload: string }
-    | { type: typeof FIRMWARE.SET_TARGET_RELEASE; payload: AcquiredDevice['firmwareRelease'] }
-    | { type: typeof FIRMWARE.RESET_REDUCER }
-    | { type: typeof FIRMWARE.ENABLE_REDUCER; payload: boolean }
-    | { type: typeof FIRMWARE.SET_INTERMEDIARY_INSTALLED; payload: boolean }
-    | { type: typeof FIRMWARE.SET_TARGET_TYPE; payload: FirmwareType }
-    | { type: typeof FIRMWARE.SET_ERROR; payload?: string }
-    | { type: typeof FIRMWARE.TOGGLE_HAS_SEED }
-    | { type: typeof FIRMWARE.REMEMBER_PREVIOUS_DEVICE; payload: Device }
-    | { type: typeof FIRMWARE.SET_IS_CUSTOM; payload: boolean }
-    | { type: typeof FIRMWARE.TOGGLE_USE_DEVKIT; payload: boolean };
-
-export const resetReducer = (): FirmwareAction => ({
-    type: FIRMWARE.RESET_REDUCER,
-});
-
-export const setStatus = (payload: AppState['firmware']['status']): FirmwareAction => ({
-    type: FIRMWARE.SET_UPDATE_STATUS,
+const MODULE_PREFIX = '@firmware';
```

Comment:
Just firmware now and will be renamed to suit the common package convention once moved.